<ui version="4.0" >
 <class>RegisterWidget</class>
 <widget class="QWidget" name="RegisterWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>552</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <property name="spacing" >
      <number>3</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabRegisters" >
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabGPR" >
        <attribute name="title" >
         <string>&amp;GPR</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3" >
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" >
           <item>
            <widget class="QTableView" name="tableGpr" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
               <horstretch>6</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font" >
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="lineWidth" >
              <number>0</number>
             </property>
             <property name="dragDropOverwriteMode" >
              <bool>false</bool>
             </property>
             <property name="dragDropMode" >
              <enum>QAbstractItemView::NoDragDrop</enum>
             </property>
             <property name="alternatingRowColors" >
              <bool>true</bool>
             </property>
             <property name="selectionMode" >
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior" >
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="showGrid" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="tableCop0" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
               <horstretch>5</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font" >
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="lineWidth" >
              <number>0</number>
             </property>
             <property name="dragDropOverwriteMode" >
              <bool>false</bool>
             </property>
             <property name="alternatingRowColors" >
              <bool>true</bool>
             </property>
             <property name="selectionMode" >
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior" >
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="showGrid" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabSpecial" >
        <attribute name="title" >
         <string>&amp;Special</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_21" >
         <item>
          <widget class="QGroupBox" name="groupBox_3" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title" >
            <string>PC</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_20" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_12" >
              <item>
               <widget class="QLineEdit" name="linePC" />
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_9" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title" >
            <string>Previous PC</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_17" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_14" >
              <item>
               <widget class="QLineEdit" name="linePrevPC" />
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_11" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="title" >
            <string>Hi / Lo</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_19" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_16" >
              <item>
               <widget class="QListView" name="listHiLo" >
                <property name="alternatingRowColors" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_10" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>3</verstretch>
            </sizepolicy>
           </property>
           <property name="title" >
            <string>Interrupt Queue</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_18" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_15" >
              <item>
               <widget class="QListView" name="listInterupt" >
                <property name="alternatingRowColors" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>148</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabCop1" >
        <attribute name="title" >
         <string>&amp;Cop1</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout" >
         <item rowspan="2" row="0" column="0" >
          <widget class="QGroupBox" name="groupBox_4" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
             <horstretch>2</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title" >
            <string>Cop1</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_11" >
            <property name="margin" >
             <number>1</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10" >
              <item>
               <widget class="QTableView" name="tableCop1" >
                <property name="font" >
                 <font>
                  <pointsize>8</pointsize>
                 </font>
                </property>
                <property name="lineWidth" >
                 <number>0</number>
                </property>
                <property name="showDropIndicator" stdset="0" >
                 <bool>false</bool>
                </property>
                <property name="alternatingRowColors" >
                 <bool>true</bool>
                </property>
                <property name="selectionMode" >
                 <enum>QAbstractItemView::SingleSelection</enum>
                </property>
                <property name="selectionBehavior" >
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
                <property name="showGrid" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QGroupBox" name="groupCop1Show" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>90</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>90</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title" >
            <string>Show as</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4" >
            <property name="spacing" >
             <number>1</number>
            </property>
            <property name="margin" >
             <number>1</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2" >
              <item>
               <widget class="QRadioButton" name="radioRawhex" >
                <property name="text" >
                 <string>&amp;Raw Hex</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioSingle" >
                <property name="text" >
                 <string>&amp;Single</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioDouble" >
                <property name="text" >
                 <string>&amp;Double</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioWord" >
                <property name="text" >
                 <string>&amp;Word</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioLong" >
                <property name="text" >
                 <string>&amp;Long</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1" >
          <spacer name="verticalSpacer_2" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>326</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabOther" >
        <attribute name="title" >
         <string>&amp;AI / VI</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6" >
         <item>
          <widget class="QGroupBox" name="groupAi" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>6</verstretch>
            </sizepolicy>
           </property>
           <property name="title" >
            <string>AI</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_9" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_10" >
              <item>
               <widget class="QListView" name="listAi" >
                <property name="alternatingRowColors" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupVi" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>15</verstretch>
            </sizepolicy>
           </property>
           <property name="title" >
            <string>VI</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_11" >
              <item>
               <widget class="QListView" name="listVi" >
                <property name="alternatingRowColors" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab" >
        <attribute name="title" >
         <string>&amp;PI / RI / SI</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout" >
         <item>
          <widget class="QGroupBox" name="groupPi" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>13</verstretch>
            </sizepolicy>
           </property>
           <property name="title" >
            <string>PI</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7" >
              <item>
               <widget class="QListView" name="listPi" >
                <property name="alternatingRowColors" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupRi" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>5</verstretch>
            </sizepolicy>
           </property>
           <property name="title" >
            <string>RI</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8" >
              <item>
               <widget class="QListView" name="listRi" >
                <property name="alternatingRowColors" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupSi" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>4</verstretch>
            </sizepolicy>
           </property>
           <property name="title" >
            <string>SI</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9" >
              <item>
               <widget class="QListView" name="listSi" >
                <property name="alternatingRowColors" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
